recursive_iterativeOddSum.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+function recursiveOddSumTo(number) {
+    let sum = 1;
+    let currentOddNumber = 1;
+    let limitNumber = number;
+    number % 2 === 0 ? limitNumber -= 1 : limitNumber -= 2;  
+    function recursiveSum(recLimitNumber) {

```

Comment:
no need to create a new function. you can check an example from yesterday lesson.

Reply:
This reminded me why I do not like recursions, they have been hard for me to implement since I first saw them. Anyway, this one is there. Frankly, I can't say I have not googled it, buy at least I understand how it works and hopefully, can recreate if needed.